lib/rubocop/cop/style/trailing_comma_in_arguments.rb

```diff
@@ -4,6 +4,13 @@ module RuboCop
   module Cop
     module Style
       # This cop checks for trailing comma in argument lists.
+      # The configuration options are:
```

Comment:
I think "The supported styles are:" would be better wording here.

Reply:
done